### PR TITLE
Fix http1 connection pool test.

### PR DESCRIPTION
**Pull request by jplevyak**
Signed-off-by: John Plevyak <jplevyak@gmail.com>

For an explanation of how to fill out the fields, please see the relevant section 
in [PULL_REQUESTS.md](./PULL_REQUESTS.md)

*Description*:
*Risk Level*:
*Testing*:
*Docs Changes*:
*Release Notes*:
[Optional Fixes #Issue]
[Optional *Deprecated*:]

### CODE REVIEW

**Comment by duderino** [comment]
@silentdai @PiotrSikora @crazyxy could one of you give this a review?

**Comment by crazyxy** [comment]
Has this been merged to upstream envoy?

File: source/common/http/http1/conn_pool.cc [hunk 1]
```diff
@@ -310,7 +310,7 @@ void ConnPoolImpl::StreamWrapper::decodeHeaders(HeaderMapPtr&& headers, bool end
       saw_close_header_ = true;
       parent_.parent_.host_->cluster().stats().upstream_cx_close_notify_.inc();
     } else if (absl::EqualsIgnoreCase(headers->Connection()->value().getStringView(),
-                               Headers::get().ConnectionValues.KeepAlive)) {
+                                      Headers::get().ConnectionValues.KeepAlive)) {
```
Comment (crazyxy) [inline source/common/http/http1/conn_pool.cc]: Revert?

Comment (jplevyak) [reply source/common/http/http1/conn_pool.cc]: This fixes the formatting.

Comment (jplevyak) [reply source/common/http/http1/conn_pool.cc]: Yes, this is going to be merged upstream.

**Pull request merged**